doc/lua_api.txt

```diff
@@ -4010,7 +4012,9 @@ Definition tables
         ^ node placement (i.e. schematics and VoxelManip) or air nodes. If
         ^ return true the node is not flooded, but on_flood callback will
         ^ most likely be called over and over again every liquid update
-        ^ interval. Default: nil ]]
+        ^ interval. Note: liquid-type nodes cannot themselves be floodable, else
```

Comment:
So the text 'else liquids would continously flood themselves and overload the update queue.' is untrue and is not needed.
Still a good idea to not set floodable on liquids, in case it is causing load without actually triggering 'on flood'.